plugins/webhooks/index.js

```diff
@@ -343,6 +345,10 @@ function pushEvent(pluginOptions, request, reply, parsed) {
                 return reply().code(204);
             }
 
+            if (ignoreUsername.includes(username)) {
```

Comment:
Let's log something like this before it's returned so that it's easier to debug.
https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/webhooks/index.js#L419